cmd/manage-crds/main.go

```diff
@@ -32,16 +32,14 @@ var logger = log.New()
 
 type config struct {
 	Debug     bool
-	crdsPaths *cli.StringSlice
+	crdsPaths *[]string
```

Comment:
This causes a `nil pointer deference` error in L107 and L119 because it's uninitialized. 

This needs to be:
```go
crdsPaths []string
```

And the corresponding flag should reference the pointer:
```go
Destination: &config.crdsPaths
```